A64-lrs-custom-metrics.md

```diff
@@ -0,0 +1,94 @@
+A64: xDS LRS Custom Metrics Support
+----
+* Author: yousukseung
+* Approver(s): markdroth
+* Status: Ready for Implementation
+* Implemented in: C-core
+* Last updated: 2023-05-10
+* Discussion at: TBA
```

Comment:
Please go ahead and create the mailing list thread and then add a link here.  This is what starts the 2-week timer before we can merge the gRFC.

Reply:
Done.

Reply:
I'll wait till 5/26.